core/src/main/java/tc/oc/pgm/api/match/event/MatchStatsEvent.java

```diff
@@ -1,19 +1,26 @@
 package tc.oc.pgm.api.match.event;
 
+import java.util.Map;
+import java.util.UUID;
 import org.bukkit.event.Cancellable;
 import org.bukkit.event.HandlerList;
 import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.stats.PlayerStats;
 
 public class MatchStatsEvent extends MatchEvent implements Cancellable {
 
   private boolean cancelled = false;
   private boolean showBest;
   private boolean showOwn;
 
-  public MatchStatsEvent(Match match, boolean showBest, boolean showOwn) {
+  private Map<UUID, PlayerStats> stats;
```

Comment:
This should not be in this event at all, stats can be obtained via getter on the stats module of the match. This event is expected to be thrown by 3rd parties to signal pgm to show the match-end stats in chat.